frontend/src/metabase/lib/formatting/time.ts

```diff
@@ -67,3 +61,35 @@ export function formatTimeWithUnit(
 
   return m.format(timeFormat);
 }
+
+interface TimeOnlyOptions {
+  local?: boolean;
+  time_enabled?: "minutes" | "milliseconds" | "seconds" | null;
+  time_format?: string;
+  time_style?: string;
+}
+
+export function formatTimeWithOptions(
```

Comment:
nit: `WithOptions` suffix may be omited since we won't have a separate `formatTime` or `formatTimeWithoutOptions`. Other formatters like `formatUrl` or `formatEmail` accept options and do not have the suffix

Reply:
Good call. Especially since this effectively replaced the old one.